src/visitors/rules/semantic/path-keys-no-trailing-slash.js

```diff
@@ -3,17 +3,13 @@ class PathKeysNoTrailingSlash {
     return 'path-keys-no-trailing-slash';
   }
 
-  OpenAPIPath() {
-    return {
-      onEnter: (node, _, ctx) => {
-        const pathLen = ctx.path.length;
-        return pathLen === 0 || ctx.path[pathLen - 1][ctx.path[pathLen - 1].length] !== '/'
-          ? null
-          : [ctx.createError(
-            'Trailing spaces in path are not recommended.', 'key',
-          )];
-      },
-    };
+  OpenAPIPath(node, _, ctx) {
+    const pathLen = ctx.path.length;
+    return pathLen === 0 || ctx.path[pathLen - 1][ctx.path[pathLen - 1].length] !== '/'
+      ? null
+      : [ctx.createError(
+        'Trailing spaces in path are not recommended.', 'key',
```

Comment:
Is this also about trailing slashes?
```suggestion
        'Trailing slashes in path are not recommended.', 'key',
```